site/assets/js/snippets.js

```diff
@@ -102,7 +102,9 @@
   // Indeterminate checkbox example in docs and StackBlitz
   document.querySelectorAll('.bd-example-indeterminate [type="checkbox"]')
     .forEach(checkbox => {
-      checkbox.indeterminate = true
+      if (checkbox.id.includes('Indeterminate')) {
```

Comment:
The other way around would be to set `checkbox.indeterminate = true` only to the first example in `.bd-example-indeterminate`:
```js
document.querySelectorAll('.bd-example-indeterminate')
  .forEach(example => {
    example.querySelector('[type="checkbox"]').indeterminate = true
  })
```
IDK what's the preferred approach here.